src/main.rs

```diff
@@ -550,6 +550,22 @@ fn install_panic_hook() {
         let mut rng = rand::thread_rng();
         let msg = msgs[rng.gen_range(0, msgs.len())];
         println!("{}", msg);
+
+        #[cfg(target_os = "linux")]
+        {
+            const SCREAM: &[u8] = include_bytes!("data/R2D2-Scream.wav");
+
+            if let Ok(mut child) = std::process::Command::new("aplay")
```

Comment:
Why use `aplay` directly instead of `rusty_audio`?

Reply:
Me guesses sacrificing cross-platformness for a low dependency count and simple, tiny code.

Reply:
I vote for `rusty_audio` so that all OS's may witness this beauty